### PR TITLE
chore: Perform perf tests on a large app

### LINKED ISSUE #9926: [Task]: Create and import golden app for performance testing
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SubTasks

- [ ] Create the golden app
- [ ] Write script to import the golden app

### LINKED ISSUE #10140: [Task]: Add a large data set for performance testing in CI
Create an app using large amount of data and pages to keep track of performance.

- [ ] Create the dummy database during setup
- [ ] Add lot of pages to catch perf regressions on entity explorer
- [ ] Add 3-4 user actions to measure performance

### PR DESCRIPTION
**Pull request by SatishGandham**
## Description
Create a moderately complex app to run performance tests on.

Adds media column to the summary table.

Fixes #10140 
Fixes #9926

## Type of change

- New feature (non-breaking change which adds functionality) 

## How Has This Been Tested?

Tested locally and this PR is a test

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

## Test coverage results :test_tube:
<details><summary>:white_circle: Total coverage has not changed</summary>
No changes to code coverage between the base branch and the head branch</details>

### CODE REVIEW

**Review by mohanarpit — APPROVED** [review]

File: app/client/perf/src/summary.js [hunk 1]
```diff
@@ -1,8 +1,10 @@
 const fs = require("fs");
 const path = require("path");
 const sd = require("node-stdev");
+var median = require("median");
 
 global.APP_ROOT = path.resolve(__dirname);
+global.APP_ROOT = "/Users/satish/work/appsmith/app/client/perf";
```
Comment (marks0351) [inline app/client/perf/src/summary.js:7]: should we have this line?

Comment (SatishGandham) [reply app/client/perf/src/summary.js:7]: Nope, will remove.

**Pull request merged**